### DISCUSSION TITLE
Support for Official Nintendo Switch Online N64 Controller

### DISCUSSION THREAD

**Discussion by QuickWick123** [nefarius/HidHide#150] [Ideas]
I attempted to use HIHIDE with NSO N64 Controller: https://www.nintendo.com/us/store/products/nintendo-64-controller/

It does not work. The controller connects via Bluetooth. it still shows up in joy.cpl

Steam continues to hijack the controller.

**Comment by nefarius** [comment]
What does "not work" mean, what is your config, screenshots, how to reproduce etc. etc.? It appears in config client UI or not? And so on.

**Comment by QuickWick123** [comment]
![image](https://github.com/nefarius/HidHide/assets/78274206/a1f20645-4c34-485f-ad82-ed9712559792)
##############################################
##############################################
##############################################
![image](https://github.com/nefarius/HidHide/assets/78274206/ff73b8eb-614a-4755-b63a-fb4f0f69762d)
##############################################
##############################################
##############################################

Steam hijacks all inputs and even with HidHide, steam still hijacks all inputs. As we can see joy.cpl shows the controller even with enable device hiding enabled and the controller selected.

**Reply by Kanuan** [reply]
1. Locate the HID game controller instance of your device in device manager
2. Open it's properties
3. Go to the details tab
4. Locate the device upper filters section
5. Send a screenshot
